content/guides/code-splitting-import.md

```diff
@@ -27,7 +27,11 @@ function determineDate() {
 
 determineDate();
 ```
-T> Keep in mind that `import()` path cannot be fully dynamic (e.g., `import(Math.random())`). Rather either completely static (e.g., `import('./locale/de.json')`) or partially static (e.g., `import('./locale/' + language + '.json')`).
+
+Note that fully dynamic statements, such as `import(foo)`, __will fail__ because webpack requires at least some file location information. This is because `foo` could potentially be any path to any file in your system or project. The `import(...)` must contain at least some information about where the module is located, so bundling can be limited to a specific directory or set of files.
```

Comment:
Two minor remarks right at the beginning:
* "split point" → "split-point" (same typesetting as in the intro above the example)
* `import(...)` → `import()`? (We've been referring to this always without arguments. But I see that here you want to stress the argument. Decide yourself. :smile:)